tests/test_v1_api.py

```diff
@@ -173,6 +173,22 @@ def test_fetch_baseline_list_sort(self):
         # check that the ascending result is the inverse of the descending result
         self.assertEqual(desc_result["data"][::-1], asc_result["data"])
 
+    def test_fetch_baseline_list_sort_updated_on(self):
+        response = self.client.get(
+            "api/system-baseline/v1/baselines?order_by=updated_on",
```

Comment:
I see updated_on here, and then modified_on mentioned elsewhere

Reply:
oops, typo.

probably not great that this passes tests, will make the test stricter.